### PR TITLE
Legal DMCA

### PR DESCRIPTION
**Pull request by eldiegod**
Issue: #498

### CODE REVIEW

File: src/views/LegalView/LegalView.style.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  />
+)
+// wrapper so we can use a specific element to render
+const _StyledText = styled(Text)``
+export const ListItem: React.FC = (props) => (
+  <li>
+    <_StyledText
+      variant="body2"
+      // secondary
+      // as="li"
```
Comment (drillprop) [inline src/views/LegalView/LegalView.style.tsx:56]:
```suggestion
```

File: src/views/LegalView/LegalView.style.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  margin-top: ${sizes(6)};
+  ol + & {
+    margin-top: 0;
+  }
+`
+
+export const Paragraph: React.FC<{ header?: boolean }> = ({ header, ...props }) => (
+  <_Paragraph
+    variant={header ? 'h6' : 'body2'}
+    // secondary={!header}
```
Comment (drillprop) [inline src/views/LegalView/LegalView.style.tsx:36]:
```suggestion
```

File: src/views/LegalView/index.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            <br />
+            Designated DMCA Agent Jsgenesis AS
+            <br />
+            CO UMA Workspace Stenersgata 8
+            <br />
+            Oslo, 0184 Norway
+            <br />
+            <br />
+            Or by telephone: +44789553019
+          </Paragraph>
```
Comment (drillprop) [inline src/views/LegalView/index.tsx:83]: A small thing, but you could use the `<address />` tag here, and add links to the phone number and email address: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

File: src/views/LegalView/LegalView.style.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+export const Box = styled.div`
+  scroll-behavior: smooth;
+  text-align: left;
+  margin-bottom: ${sizes(10)};
+  position: relative;
+  height: 85vh;
+  width: 100%;
+  background-color: ${colors.gray[800]};
+  overflow: auto;
```
Comment (kdembler) [inline src/views/LegalView/LegalView.style.tsx:14]: There are 2 scroll containers in the view, one is this Box, second is the `body` itself. Once you stop scrolling the box, the body will scroll. Maybe we should just make the Box grow and let the user scroll the body?

File: src/views/LegalView/LegalView.style.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  }
+`
+const _Paragraph = styled(Text)`
+  margin-top: ${sizes(6)};
+  ol + & {
+    margin-top: 0;
+  }
+`
+
+export const Paragraph: React.FC<{ header?: boolean }> = ({ header, ...props }) => (
```
Comment (kdembler) [inline src/views/LegalView/LegalView.style.tsx:33]: We need to adjust colors for the text. The body text is too dark for the background, the contrast is too low. In terms step we use `secondary` for everything except the header

File: src/views/LegalView/LegalView.style.tsx (hunk 1 above)
Comment (eldiegod) [reply src/views/LegalView/LegalView.style.tsx:56]: Left this as comments so when we merge to summer we can uncomment those as they don't work on master, should have mentioned in the PR 😜

**Review by kdembler — APPROVED** [review]

**Pull request merged**